spec/examples/full_email.html

```diff
@@ -79,7 +79,7 @@ <h2 style="font-family: Times New Roman; font-size: 1.8; font-weight: normal; li
   <h3 style="font-family: Times New Roman; font-size: 18px; font-weight: normal; line-height: 2em; margin: 10px 0 0">Your achievements</h3>
   <table class="current-usage with-icon-left" style="border-collapse: collapse; border-spacing: 0; margin-bottom: 20px; margin-top: 20px; width: 100%">
     <tr>
-      <th style="border: none; font-family: Times New Roman; font-size: 14px; font-weight: bold; margin: 0; padding: 0; text-align: left; vertical-align: middle; width: 50px" align="left" valign="middle"><img src="test.png"></th>
+      <th style="border: none; font-family: Times New Roman; font-size: 14px; font-weight: bold; margin: 0; padding: 0; text-align: left; vertical-align: middle; width: 50px" align="left" valign="middle"></th>
```

Comment:
Shouldn't this stay as-is, with the expected value updated to include `[image]`?

Reply:
Sure I can do that. My thought was that we had enough tests at work here so I took some liberties.

Reply:
Ah, yes, after reviewing the rest of the pull, I agree.